docs/dev_guides/core_concepts.md

```diff
@@ -0,0 +1,44 @@
+# Core concepts
+
+This package provides reusable components (*`threedee` objects*) which 
+- enable the use of certain custom 3D interactive functionality
+- simplify the development of custom 3D interactive functionality
+
+Broadly, *`threedee` objects* can be split into two categories, **manipulators** and **annotators**.
+
+## Manipulators
+A ```Manipulator``` is an object at a specific position in the scene made up of 
```

Comment:
```suggestion
A ```Manipulator``` is an object used to set the position of visuals in the scene. A ```Manipulator``` is made up of 
```

Reply:
unclear to me what 'visuals' are in this context... will rework a bit from my side and ping for re-review when ready :)

Reply:
I wanted to say "objects", but we've already used it to describe threedee objects 😉